doc/Makefile

```diff
@@ -15,10 +17,13 @@ overview-slides.html : overview-slides.md
 	  # # -V minScale=2 \
 	  # # -V maxScale=2
 
+md:
+	./convert.sh $(SRC) $(OUT)
```

Comment:
You didn't commit that file yet, it seems

Reply:
Done.